core/tests/puppeteer-acceptance-tests/spec/blog-admin-tests.spec.js

```diff
@@ -0,0 +1,124 @@
+// Copyright 2023 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/** TODO(#16552): Write a test to check that the user is unable to publish
```

Comment:
These TODOs should be placed near the lines of code they pertain to.

Reply:
@Shivkant-Chauhan has a PR in place for that #17354 . I plan to merge this before that so he can do the necessary edits to it.

Reply:
Sure, but my point about where the TODOs are to be placed in the code still stands. This place is not the right one.

Reply:
@ashish-patwal did you see this comment? It hasn't been responded to.

Reply:
@seanlip Sorry ! Forgot to update here. I have actually done the adjustments. You can have a look